tests/block_ordering.php

```diff
@@ -1,118 +0,0 @@
-	array("block_order" => "C", "item_order" => 1, "id" => "xC2"),
-	array("block_order" => "", "item_order" => 5, "id" => "xend"),
-	
-) );
-
-echo "
----------
-";
-blocksort( array(
-	array("block_order" => "A", "item_order" => 1, "id" => "xA1"),
-	array("block_order" => "A", "item_order" => 2, "id" => "xA2"),
-	array("block_order" => "B", "item_order" => 1, "id" => "xB1"),
-	array("block_order" => "B", "item_order" => 1, "id" => "xB2"),
-	array("block_order" => "B", "item_order" => 2, "id" => "xB3"),
-	array("block_order" => "C", "item_order" => 1, "id" => "xC1"),
-	array("block_order" => "C", "item_order" => 1, "id" => "xC2"),
-	array("block_order" => "C", "item_order" => 1, "id" => "xC3"),
-	
-));
-echo "
----------
-";
-
-blocksort( array(
-	array("block_order" => "A", "item_order" => 1, "id" => "A"),
-	array("block_order" => "B", "item_order" => 1, "id" => "B"),
-	array("block_order" => "C", "item_order" => 1, "id" => "C"),
-	array("block_order" => "D", "item_order" => 1, "id" => "D"),
-	array("block_order" => "E", "item_order" => 1, "id" => "E"),
-	array("block_order" => "F", "item_order" => 1, "id" => "F"),
-	array("block_order" => "G", "item_order" => 1, "id" => "G"),
-	array("block_order" => "H", "item_order" => 1, "id" => "H"),
-));
-echo "
----------
-";
-blocksort( array(
-	array("block_order" => "", "item_order" => 1, "id" => "x1"),
-	array("block_order" => "", "item_order" => 2, "id" => "x2"),
-	array("block_order" => "", "item_order" => 3, "id" => "x3"),
-	array("block_order" => "", "item_order" => 4, "id" => "x4"),
-	array("block_order" => "", "item_order" => 5, "id" => "x5"),
-	array("block_order" => "", "item_order" => 6, "id" => "x6"),
-	array("block_order" => "", "item_order" => 7, "id" => "x7"),
-	array("block_order" => "", "item_order" => 8, "id" => "x8"),
-));
-echo "
----------
-";
-blocksort( array(
-	array("block_order" => "", "item_order" => 1, "id" => "x1"),
-	array("block_order" => "", "item_order" => 1, "id" => "x2"),
-	array("block_order" => "", "item_order" => 1, "id" => "x3"),
-	array("block_order" => "", "item_order" => 1, "id" => "x4"),
-	array("block_order" => "", "item_order" => 1, "id" => "x5"),
-	array("block_order" => "", "item_order" => 1, "id" => "x6"),
-	array("block_order" => "", "item_order" => 1, "id" => "x7"),
-	array("block_order" => "", "item_order" => 1, "id" => "x8"),
-));
-echo "
----------
-";
-
-blocksort( array(
-	array("block_order" => "A", "item_order" => 1, "id" => "xA1"),
-	array("block_order" => "A", "item_order" => 1, "id" => "xA2"),
-	array("block_order" => "B", "item_order" => 1, "id" => "xB1"),
-	array("block_order" => "B", "item_order" => 1, "id" => "xB2"),
-	array("block_order" => "", "item_order" => 1, "id" => "xnoblock"),
-	array("block_order" => "*", "item_order" => 1, "id" => "x*"),
-	array("block_order" => "*", "item_order" => 1, "id" => "x*"),
-	array("block_order" => "&", "item_order" => 1, "id" => "x&"),
-	array("block_order" => "&", "item_order" => 1, "id" => "x&"),
-));
```

Comment:
Maybe this could be converted to a proper unit test instead of completely removing it?

Reply:
If you explain the sense to me ;) I really didn't get what this code is for.

Reply:
I think Ruben's intention was to be able to have "groups" of items and be able to define an order for those groups, an under each group an order for items. So in the items sheet, you add a column called block_order and the various unique values in this column will define the various groups. However in the unit test now this will me mocked with a dataProvider. Hopefully I explained somewhat clearly

Reply:
I restored the file for now. I don't see why we need this anymore, but if you want to keep it, we do. Ready to merge.